cmd/kcp-front-proxy/options/options.go

```diff
@@ -17,27 +17,62 @@ limitations under the License.
 package options
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
 
 	"github.com/spf13/pflag"
 
+	"k8s.io/apiserver/pkg/authentication/request/x509"
+	genericapiserver "k8s.io/apiserver/pkg/server"
 	apiserveroptions "k8s.io/apiserver/pkg/server/options"
 
 	proxyoptions "github.com/kcp-dev/kcp/pkg/proxy/options"
 )
 
-type Options struct {
-	SecureServing *apiserveroptions.SecureServingOptionsWithLoopback
-	Proxy         proxyoptions.Options
+// ClientCertOptions wraps ClientCertAuthenticationOptions so we don't pull in
+// more auth machinery than we need with DelegatingAuthenticationOptions
```

Comment:
just to be clear: we definitely do not want full authentiaction, e.g. oidc or similar here. JWT should just passed on.

Reply:
Right - I looked at delegating auth first and realized it pulled in too much. So I decided to use the ClientCertAuthenticationOptions instead.